app/src/main/java/com/ml/shubham0204/docqa/ui/screens/docs/DocsScreen.kt

```diff
@@ -217,24 +224,29 @@ private fun DocOperations(docsViewModel: DocsViewModel) {
             }
         }
 
-    Row(
+    Column(
```

Comment:
We should place the buttons in a `Row`. Placing them in a `Column` occupies more screen space, with the space to the right of the buttons being unused.